labgrid/driver/qemudriver.py

```diff
@@ -168,6 +168,11 @@ def get_qemu_base_args(self):
                 cmd.append(
                     f"if=virtio,format={disk_format},file={disk_path}{disk_opts}")
                 boot_args.append("root=/dev/vda rootwait")
+            elif self.machine == "virt":
+                cmd.append("-drive")
+                cmd.append(
+                    f"if=virtio,format={disk_format},file={disk_path}{disk_opts}")
+                boot_args.append("root=/dev/vda rootwait")
```

Comment:
As far as I can see, this is exactly what we do for the "pc" case. Do you expect any differences in the future from "pc"? If not, we could handle "pc" and "virt" in the same clause.

Reply:
Okay, let's keep it that way for now. We can always create a clean up PR on  top.